data/schemas/node-type-genspec.schema

```diff
@@ -9,6 +9,13 @@
       ]
     },
 
+    "uint_or_limit": {
+      "oneOf": [
+        { "type": "integer" },
```

Comment:
On 11-05-2016 17:12, Gustavo Sverzut Barbieri wrote:
> In data/schemas/node-type-genspec.schema
> <https://github.com/solettaproject/soletta/pull/2017#discussion_r62917901>:
>
>> @@ -9,6 +9,13 @@
>>        ]
>>      },
>>
>> +    "uint_or_limit": {
>> +      "oneOf": [
>> +        { "type": "integer" },
>
> is there any way to say ">=0"?

Hmmm... now that you told about it, I think there's a minimum attribute. 
Will check it.

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/2017/files/fd10e6713bb2394b5c0e44cb1c50a843b7185efd#r62917901>
>